packages/bot/src/stores/quick-strategy-store.js

```diff
@@ -2,33 +2,41 @@ import {
     observable,
     action,
     runInAction,
-}                from 'mobx';
-import ApiHelper from '../services/api/api-helpers';
-import config    from '../constants/index';
-import { load }  from '../scratch/utils';
+}                    from 'mobx';
+import ApiHelper     from '../services/api/api-helpers';
```

Comment:
We use plural `ApiHelpers` across the app.

```suggestion
import ApiHelpers     from '../services/api/api-helpers';
```